src/ImageProcessor.Plugins.Cair/Imaging/ContentAwareResizeLayer.cs

```diff
@@ -88,22 +88,15 @@ public OutputType OutputType
         public int Timeout { get; set; } = 60000;
 
         /// <summary>
-        /// Returns a value that indicates whether the specified object is an 
-        /// <see cref="ContentAwareResizeLayer"/> object that is equivalent to 
-        /// this <see cref="ContentAwareResizeLayer"/> object.
+        /// Determines whether the specified <see cref="System.Object" />, is equal to this instance.
         /// </summary>
-        /// <param name="obj">
-        /// The object to test.
-        /// </param>
+        /// <param name="obj">The <see cref="System.Object" /> to compare with this instance.</param>
         /// <returns>
-        /// True if the given object  is an <see cref="ContentAwareResizeLayer"/> object that is equivalent to 
-        /// this <see cref="ContentAwareResizeLayer"/> object; otherwise, false.
+        ///   <c>true</c> if the specified <see cref="System.Object" /> is equal to this instance; otherwise, <c>false</c>.
         /// </returns>
         public override bool Equals(object obj)
```

Comment:
I just use `/// <inheritdoc/>` for this kind of stuff now. It works well as of recent VS versions.

Reply:
Changing the XML documentation to use `<inheritdoc />` might be something for another PR, so we don't need to block this one.